Makefile

```diff
@@ -5,9 +5,24 @@ help:
 	@# https://stackoverflow.com/a/35730928
 	@awk '/^#/{c=substr($$0,3);next}c&&/^[[:alpha:]][[:alnum:]_-]+:/{print substr($$1,1,index($$1,":")),c}1{c=0}' Makefile | column -s: -t
 
+.PHONY: required-build-tools
+expect-tools:
```

Comment:
nice!

is this compatible with windows? or could we skip if windows

Reply:
You bring up a good point, do you expect contributors to dev/test outside of Darwin/Linux? Windows devs can always use WSL. The make helper can probably won't work outside of Mac/Linux given "which" is a requirement for checking existence of a tool. We can gate the check for Mac/Linux host and leave windows users to figure it out themselves.